mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -78,8 +77,15 @@ def __init__(self,
         self._start_with_defaults: bool = bool(
             strtobool(str(self._config.pop('start_with_defaults', True))))
         self._max_iter = int(self._config.pop('max_suggestions', 100))
-        self._opt_target = str(self._config.pop('optimization_target', 'score'))
-        self._opt_sign = {"min": 1, "max": -1}[self._config.pop('optimization_direction', 'min')]
+
+        opt_targets: Dict[str, str] = self._config.pop('optimization_target', {'score': 'min'})
```

Comment:
```suggestion
        opt_targets: Dict[str, str] = self._config.pop('optimization_targets', {'score': 'min'})
```

For consistency and make the config a little more self documenting.